h/static/styles/partials-v2/_search.scss

```diff
@@ -0,0 +1,91 @@
+// Search page
+// -----------
+// Styles for the search page which displays the results
+// of activity queries.
+
+.search {
+  font-size: $normal-font-size;
+  margin-top: 20px;
+  color: $grey-6;
+}
+
+.search-results {
+  list-style: none;
+  width: 950px;
+}
+
+.search-result {
+  padding: 10px 0px;
+  border-bottom: 1px solid $grey-2;
+}
+
+.search-result__details {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  padding-left: 0px;
+
+  &:hover {
+    background-color: $grey-2;
+  }
+}
+
+.search-result__timeframe {
+  font-weight: bold;
+  padding-bottom: 10px;
+  border-bottom: 1px solid $grey-2;
+  margin-top: 30px;
+}
+
+.search-result__domain {
+  width: 130px;
+  text-indent: 10px;
+  color: $grey-4;
+}
+
+.search-result__title {
+  font-weight: bold;
+  width: 695px;
```

Comment:
Be careful about using large fixed widths as these are likely to break on mobile